buzzmobile/include/constants.py

```diff
@@ -1,5 +1,5 @@
-image_width = 800
-image_height = 800
-pixels_per_m = 100
+image_width = 1200
```

Comment:
Since we're moving all constants to the `.yaml` file, do we still need this file/folder?